auth_totp/README.rst

```diff
@@ -0,0 +1,103 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===========
+MFA Support
+===========
+
+This module adds support for MFA using TOTP (time-based, one-time passwords). 
```

Comment:
Break this up. Try to max out around 2-3 sentences per paragraph when tech writing.

Get with me in chat at some point and we can go over some points. Preferably next week after SMD's newb @kellylougheed starts, so we can kill multiple explanations in the same stone.